### PR TITLE
[WIP] Writing Custom Images + Configure examples

### PR DESCRIPTION
**Pull request by khorne3**
Related to https://github.com/cdr/docs/pull/94

### CODE REVIEW

**Comment by khorne3** [comment]
@bpmct somehow I didn't track your branch properly 🙃 but here's what I was thinking -- writing images should probably go under images, and the remaining examples I moved to configure. WDYT?

File: images/configure.md [hunk 1]
```diff
@@ -7,20 +7,25 @@ If you have configuration instructions that apply to everyone who uses a given
 image to create environments, you can define them using the **/coder/configure**
 file.
 
-For example, you might want all of the image's users to check for the presence
-of the project's Git repo (and if it's not there, clone it).
+You can use the configure script to:
+
+- Run [Coder CLI](https://github.com/cdr/coder-cli) commands
+- Check for and clone a GitHub repo if it isn't present
+- Install global npm packages
```
Comment (bpmct) [inline images/configure.md:14]:
```suggestion
```

I realized we should take this one out since it is language-specific

**Review by bpmct — APPROVED** [review]
this is awesome, thanks!

**Comment by khorne3** [comment]
@bpmct Going to merge this PR and close https://github.com/cdr/docs/pull/94 if you have no objections?

**Comment by bpmct** [comment]
> @bpmct Going to merge this PR and close #94 if you have no objections?

No objections here. Thanks again Katie

**Pull request merged**